src/Client.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-var APIVersion = '2.7'
+var APIVersion = '3'
```

Comment:
In the future, API version bumps can be their own PRs.

Reply:
@nathaniel-may In order to get our tests to pass, I need to use the `3` header or else the Core throws an error 🙏

Reply:
right. what I mean is that we can issue a PR with this line only then start feature work. It's a minor point.

Reply:
@nathaniel-may Ah, okay, I misunderstood. Apologies!